lib/decision_review/service.rb

```diff
@@ -75,7 +75,7 @@ def get_contestable_issues(user)
 
     def request_headers(user)
       {
-        'veteranId' => user.ssn,
+        'x-va-ssn' => user.ssn,
         'receiptDate' => Time.current.strftime('%Y-%m-%d')
```

Comment:
```suggestion
        'x-va-receipt-date' => Time.current.strftime('%Y-%m-%d')
```

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/623e4284fdb573fc11bd3de18abfd563928748fa/modules/appeals_api/app/swagger/appeals_api/v1/contestable_issues_controller_swagger.rb#L19

Reply:
you could also use `Time.current.strftime('%F')`